### ISSUE TITLE
NVDA fails to read UAC (User Account Control)

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#19216]
### Brief summary

I can't say exactly from when, but I've noticed these days that NVDA stopped reading the UAC (User Account Control) windows. I haven't checked, but I think the same is true for the Windows login screen.

The "Use NVDA during sign-in (requires administrator privileges)" option is checked. I even unchecked it, applied the settings, checked the option back and again applied the settings. It did not help.

Based on my visual observations, I think NVDA is not loading at all on those screens (visual highlight should be displayed even on those screens but it is not).

I'm running the latest alpha build of NVDA on a machine with Windows 11 version 25H2 (I'll provide more details of the test environment in the respective field below).

UAC (User Account Control) is set to 67%, which I think is the default value.

### Steps to reproduce

1. Launch NVDA.
2. Launch a program with administrative privileges.
3. Observe if NVDA reads the elements in the displayed UAC (User Account Control) window.

### Actual behavior

NVDA behaves as if it is not loaded (e.g. does not read the elements in the UAC window).

### Expected behavior

NVDA correctly reads the elements in the UAC window.

### NVDA type

installed copy

### NVDA version

alpha-53423,57f2e9dc (2026.1.0.53423)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

No, because I don't (and I can't) have any other versions of NVDA installed on the machine.

### Windows version

Windows 11 Pro version 25H2 (build 26200.7171), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by tareh7z** [comment]
Hi!

I can confirm that this issue is reproducible on the same Windows and NVDA versions.

For now, the only workaround I found is using the keyboard shortcuts for confirming or dismissing the UAC prompt (Yes/No), i.e., `Alt+Y` and `Alt+N` on the English locale.

Thanks.

**Comment by XLTechie** [comment]
Just to be very sure: it was working in the prior alpha?

**Comment by k-kolev1985** [comment]
I'm not sure to be honest in which alpha it started occurring, but from my observations, it is from fairly recently

**Comment by XLTechie** [comment]
Have you tried the "Use currently saved settings during sign-in and on secure screens" button in general settings?

It is possible that something about your old configuration (the one saved in the administrative copy) is not compatible with some change made to your system.
For example in the administrative copy of NVDA, one of your add-ons may not be updated, and may be crashing NVDA or preventing speech or similar.

**Comment by tareh7z** [comment]
Luke wrote:

> Have you tried the "Use currently saved settings during sign-in and on secure screens" button in general settings?
> 
> It is possible that something about your old configuration (the one saved in the administrative copy) is not compatible with some change made to your system.
> For example in the administrative copy of NVDA, one of your add-ons may not be updated, and may be crashing NVDA or preventing speech or similar.

Yes, I always activate the "Use currently saved settings during sign-in and on secure screens" option after each update, just to make sure both copies are fully up to date.

For reference, Narrator does not encounter this issue.

**Comment by k-kolev1985** [comment]
To confirm the observations by @hdzrvcc0X74 - I copied the **nvda.ini** file from my personal directory to NVDA's **\\Program Files\NVDA\systemConfig\\** directory, but the result was the same, even after restarting NVDA. I don't have any add-ons in the **\\systemConfig\addons\\** directory of NVDA. And for me **Narrator** also works as expected on those screens.

**Comment by tareh7z** [comment]
Hi @k-kolev1985!

Just copying the `nvda.ini` file by itself isn’t enough to apply the settings to the system configuration. The proper way is to open the general NVDA settings with `NVDA+Control+G` and activate **Use currently saved settings during sign-in and on secure screens (requires administrator privileges)** option.

**Comment by derekriemer** [comment]
I'm also having this issue. is there a way to pull logs for the system copy of NVDA? there was at one point a registry key that let some debugging occur but not sure if that's still present.

**Comment by SaschaCowley** [comment]
@derekriemer yes, the `serviceDebug` regkey can be used to enable logging on secure screens. Please ensure you delete the key as soon as you are finished with it as having it set is a massive security risk

**Comment by SaschaCowley** [comment]
I'm facing this too. Pending more formal triage, I'm labelling this P1 and release blocking. Putting on the 2025.3.2 milestone in case the regression was introduced by the security patch in that version - if it wasn't, I will move to 2026.1.
